src/gui/DatabaseOpenWidget.cpp

```diff
@@ -400,45 +400,62 @@ void DatabaseOpenWidget::handleKeyFileComboChanged()
     m_ui->keyFileClearIcon->setVisible(m_keyFileComboEdited);
 }
 
-void DatabaseOpenWidget::pollYubikey()
+void DatabaseOpenWidget::pollHardwareKey()
 {
+    if (m_yubiKeyBeingPolled) {
+        return;
+    }
+
+    m_ui->comboChallengeResponse->clear();
+    m_ui->comboChallengeResponse->addItem(tr("Detecting Hardware Keys..."));
```

Comment:
Should be a real ellipsis, not three dots.

Reply:
We have several places in the code where we should convert to ellipses.